build/build.css

```diff
@@ -753,6 +753,34 @@ hr:nth-of-type(6) {
   background-image: url("metalsmith.io/images/divider-6.png");
 }
 
+hr:nth-of-type(7) {
+  background-image: url("metalsmith.io/images/divider-1.png");
+}
+
```

Comment:
Yeah, I think the `build/` should be removed. I just added to the static-site example in metalsmith and suggested to gitignore build:
[metalsmith PR #237](https://github.com/metalsmith/metalsmith/pull/237/commits/51a8f80c6a3871f65e0c58ec8f47525e051c0ee2)